### PR TITLE
Issue #244 Fixed an error in a link and added some more movies

### PR DESCRIPTION
**Pull request by damaru2**
Sorry this took so long. I had forgotten about this

### CODE REVIEW

**Comment by RichardLitt** [comment]
Thanks @damaru2! Can you add your signature to the reviews? 

```
 - [@damaru2](https://github.com/damaru2)
```

**Comment by damaru2** [comment]
Done

File: readme.md [hunk 1]
```diff
@@ -1468,7 +1468,7 @@ One of the most well known scifi films, this is a great expose on what people th
 
 Categories: `remake` `steampunk`
 
-> Hoping to alter the events of the past, a 19th century inventor instead travels 800,000 years into the future, where he finds humankind divided into two warring races.
+> Hoping to alter the events of the past, a 19th century inventor instead travels 800,000 years into the future, where he finds humankind divided into two warring races. - [@damaru2](https://github.com/damaru2)
```
Comment (RichardLitt) [inline readme.md]: This is the description, not a comment for this movie - not sure the link is right here.

File: readme.md [hunk 2]
```diff
@@ -1117,7 +1117,7 @@ Absolutely beautiful film. The cinematography is fantastic; Michael Caine has a
 
 Categories: `drama` `thriller`
 
-Several different independent stories told beautifully, each of them set in a different era.
+Several different independent stories told beautifully, each of them set in a different era. - [@damaru2](https://github.com/damaru2)
```
Comment (RichardLitt) [inline readme.md]: What did you think, besides 'beautifully'?

File: readme.md [hunk 3]
```diff
@@ -1204,7 +1204,7 @@ Kind of a parody about Google. The owner of a huge search provider creates an A.
 
 Categories: `thriller`
 
-An independent movie showing how humans react and interact with the technologies that surround them, in this case, the world of video games in perfect virtual reality.
+An independent movie showing how humans react and interact with the technologies that surround them, in this case, the world of video games in perfect virtual reality. - [@damaru2](https://github.com/damaru2)
```
Comment (RichardLitt) [inline readme.md]: What did you think?

File: readme.md [hunk 4]
```diff
@@ -1322,7 +1322,7 @@ This is one of the best drawn-out psychodrama films about space out there, along
 
 Categories: `drama` `romance`
 
-This is a tale about choice. We see a glimpse of the infinite possibilities of the protagonist's life. The film also makes substantial use of chaos theory, string theory, and the butterfly effect to accentuate the lack of control that humanity as individuals possesses.  It makes substantial use of chaos theory, string theory, and the butterfly effect to accentuate the lack of control that humanity as individuals possesses.
+This is a tale about choice. We see a glimpse of the infinite possibilities of the protagonist's life. A myriad of stories are told, each of them a different possible life Mr. Nobody could have lived. The film also makes substantial use of chaos theory, string theory, and the butterfly effect to accentuate the lack of control that humanity as individuals possesses - [@damaru2](https://github.com/damaru2)
```
Comment (RichardLitt) [inline readme.md]: What did you think?

**Comment by damaru2** [comment]
I see, you wanted a more personal review. I have changed everything but the review in cloud atlas which I'll write later. I hope now it's better

**Comment by RichardLitt** [comment]
This looks great. (Cloud Atlas is also one of _my_ favourite movies; didn't realise it wasn't in this list!). 

If you could add a review for The Time Machine, this is ready to go!

**Comment by RichardLitt** [comment]
Fantastic. Thank you!

**Pull request merged**